pkg/report/writer.go

```diff
@@ -15,7 +15,7 @@ import (
 	"github.com/olekukonko/tablewriter"
 )
 
-type Results []Result
+type Results []*Result
```

Comment:
I couldn't reason out why this was being changed to a `[]*Result`. Would it make sense to change it back to `[]Result` unless I've missed a reason not to, in order to keep things simple?

Reply:
One thought: https://github.com/golang/go/wiki/CodeReviewComments#pass-values

Reply:
Based on our conversation offline, I've switched it back to `[]Result` in this commit https://github.com/aquasecurity/trivy/pull/201/commits/ea7f9e058c83b8683e5596736832cc506fe1bb69